src/test/java/org/apache/commons/lang3/RandomStringUtilsTest.java

```diff
@@ -16,18 +16,18 @@
  */
 package org.apache.commons.lang3;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertTrue;
-import static org.junit.Assert.fail;
+import org.junit.Test;
```

Comment:
yeah sorry IDE did that, I decided to not correct it whilst waiting to see if this would be functionality the project would accept.